Lib/types.py

```diff
@@ -143,6 +143,28 @@ def _calculate_meta(meta, bases):
                         "of the metaclasses of all its bases")
     return winner
 
+
+def get_orig_bases(tp, /) -> tuple[type, ...] | None:
+    """Get the __orig_bases__ (see PEP 560) of a class.
```

Comment:
We shouldn't refer users to PEP 560 here, because PEP 560 contains this paragraph:

> NOTE: These two method names are reserved for use by the typing module and the generic types machinery, and any other use is discouraged. The reference implementation (with tests) can be found in [[4]](https://peps.python.org/pep-0560/#id10), and the proposal was originally posted and discussed on the typing tracker, see [[5]](https://peps.python.org/pep-0560/#id11).

The "two method names" being referred to here are `__mro_entries__` and `__orig_bases__`.

In fact, we shouldn't mention the `__orig_bases__` attribute at all, since the whole point of this function is to provide an introspection API for users while making sure that the `__orig_bases__` attribute itself remains an implementation detail of the `typing` system that is not exposed as public API.

You'll need to find a way of describing what this function returns without referring to the __orig_bases__ attribute or PEP 560.